### DISCUSSION TITLE
Ripristino password amministratore

### DISCUSSION THREAD

**Discussion by andreazanardo2002** [iisgiua/giuaschool#344] [Richieste di aiuto]
Salve, come faccio a ripristinare la password dell'utente amministratore? Non riesco a effettuare il login

**Comment by trinko** [comment]
Nella 1.5.0 non c'è possibilità di fare il recupero password per l'amministratore. La modifica è stata già inserita nelle modifiche successive e sarà nella 1.5.1.

Attualmente, l'unico modo è agire direttamente sul database con PHPMyAdmin o, se non hai dati importanti, reinstallare da capo.

Usando PHPMYAdmin, devi andare sulla pagia di inserimento SQL, inserire ed eseguire il comando seguente:
```
UPDATE gs_utente SET password='$2y$13$JwJygXTuNXYDUqMIV7B8KuOWOS8QHf3ZT9i.4Xa7QOg1hVMl9im8K' WHERE username='admin';
```
Questo dovrebbe permetterti l'accesso usando come password **12345678**

**Comment by ghost** [comment]
@andreazanardo2002 Buongiorno! Avrei bisogno di parlarti, in merito a questa richiesta. Posso parlarti in privato? Mi scrivi alla mia mail: padovesi.roberto@gmail.com? Grazie!
